server/schemas/resolvers.js

```diff
@@ -115,6 +160,14 @@ const resolvers = {
         );
       }
       throw new AuthenticationError('You need to be logged in!');
+    },
+
+    removeProfileFromProject: async (parent, { projectId, profileId }) => {
+      return Project.findOneAndUpdate(
+        { _id: projectId },
+        { $pull: { profile: profileId } },
```

Comment:
That is so cool! I didn't know about $pull but it makes so much sense for what you're trying to do